.github/workflows/release-docker-images.yml

```diff
@@ -0,0 +1,48 @@
+name: Monstache release docker images
+
+on:
+  release:
+    types: [created]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+permissions:
+    contents: write
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
```

Comment:
Sorry to be a pain but can we lock all the `docker` action to the latest commit SHA for each?  
* metadata-action@v4
* setup-qemu-action@v2
* setup-buildx-action@v2
* login-action@v2
* build-push-action@v4

Reply:
No worry I just forgot to do it when you mentionned it in the last PR.